quic/s2n-quic-core/src/datagram/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod default;
+pub mod unreliable_datagram;
```

Comment:
This module name seems a bit redundant to me. Isn't it just the traits? I would do something like:
```suggestion
mod traits;
pub use traits::*;
```

Reply:
Thanks! I struggle with names!